src/account-mgr.cpp

```diff
@@ -207,3 +231,36 @@ Account AccountManager::getAccountBySignature(const QString& account_sig) const
 
     return Account();
 }
+
+bool AccountManager::clearAccountToken(const Account& account)
+{
+    int i = 0;
```

Comment:
What about using `size_t` or `unsigned` here?

`std::vector<T>::size()` returns a `size_t` here, which is unsigned.